### PR TITLE
feat(core/Executions): Make component overridable

### PR DESCRIPTION
**Pull request by bbrina**
## What does this PR do?
Adds code for the Executions component to be overridable

### CODE REVIEW

**Review by dogonthehorizon — APPROVED** [review]

File: app/scripts/modules/core/src/pipeline/executions/Executions.tsx [hunk 1]
```diff
@@ -447,3 +449,5 @@ export class Executions extends React.Component<IExecutionsProps, IExecutionsSta
     return null;
   }
 }
+
+export default Executions;
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/pipeline/executions/Executions.tsx:453]: Deck doesn't use default exports, please remove this.
```suggestion
```

File: app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/TitusMapLayout.less [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .TitusMapLayout_Contents * {
-  flex: auto
+  flex: auto;
```
Comment (christopherthielen) [inline app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/TitusMapLayout.less:2]: please revert

File: app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/ServerGroupParameters.tsx [hunk 3]
```diff
@@ -1,25 +1,8 @@
...
-  IWizardPageComponent,
-  LayoutProvider,
-  MapEditorInput,
-  PlatformHealthOverrideInput,
-  robotToHuman,
-  SelectInput,
-  TextInput,
-  useFormInputValueMapper,
-} from '@spinnaker/core';
+import { AccountTag, Application, CheckboxInput, ChecklistInput, FormikFormField, FormValidator, HelpField, IFormInputProps, IWizardPageComponent, LayoutProvider, MapEditorInput, PlatformHealthOverrideInput, robotToHuman, SelectInput, TextInput, useFormInputValueMapper } from '@spinnaker/core';
```
Comment (christopherthielen) [inline app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/ServerGroupParameters.tsx:5]: these two other changes by the auto-formatter or linter are unrelated to this PR, please remove them (feel free to file a separate PR if you want)

File: app/scripts/modules/core/src/pipeline/executions/Executions.tsx [hunk 4]
```diff
@@ -44,6 +45,7 @@ const forwardedExecutions = new Set();
 // This ensures we only forward to permalink on landing, not on future refreshes
 let disableForwarding = false;
 
+@Overridable('createExecutions')
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/pipeline/executions/Executions.tsx:48]:
```suggestion
@Overridable('PipelineExecutions')
```

**Review by christopherthielen — CHANGES_REQUESTED** [review]

**Comment by christopherthielen** [comment]
@bbrina we can change the failing unit test to look for the spinner instead of inspecting component state.

```jsx
  it('should not set loading flag to false until executions and pipeline configs have been loaded', (done) => {
    initializeApplication();
    expect(component.find(Spinner).length).toBe(1);
    application.executions.dataUpdated();
    application.pipelineConfigs.dataUpdated();
    scope.$digest();
    setTimeout(() => {
      component.setProps({});
      expect(component.find(Spinner).length).toBe(0);
      done();
    }, 100);
  });
```

**Comment by bbrina** [comment]
Thanks @christopherthielen!

**Review by christopherthielen — APPROVED** [review]

**Pull request merged**